src/Compiler/Checking/NameResolution.fsi

```diff
@@ -684,6 +684,7 @@ val internal ResolveLongIdentAsModuleOrNamespace:
     id: Ident ->
     rest: Ident list ->
     isOpenDecl: bool ->
+    notifySink: bool ->
```

Comment:
I'd say having two bools nearby is confusing, so suggest having a proper DU type instead. But not a big deal.

Reply:
Yeah. I already don't like we have to propagate the argument. It could be also solved by using a separate function, but then what would it be called? `ResolveLongIdentAsModuleOrNamespaceDontNotifySink` 🤔

Reply:
Well better verbose than sorry :D

Reply:
I tried to weigh what's the least ugly and went with the DU.